docs/API-REFERENCE.md

```diff
@@ -1 +1,2640 @@
+
+  ... // usage of string_value
+
+  jerry_release_value (string_value);
+}
+```
+
+**See also**
+
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_create_error_sz
+
+**Summary**
+
+Creates an error object. Caller should release the object with jerry_release_object, just when
+the value becomes unnecessary.
+
+**Prototype**
+
+```c
+jerry_object_t *
+jerry_create_error_sz (jerry_error_t error_type,
+                       const jerry_char_t *message_p,
+                       jerry_size_t message_size);
+```
+
+- `error_type` - type of the error.
+- `message_p` - value of 'message' property of the constructed error object.
+- `message_size` - size of the message in bytes.
+- return value - pointer to the created error object.
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  jerry_char_t message[] = "error";
+  jerry_object_t *error_obj = jerry_create_error_sz (JERRY_ERROR_COMMON,
+                                                     message,
+                                                     strlen ((char *) message));
+
+  ... // usage of error_obj
+
+  jerry_release_object (error_obj);
+}
+```
+
+**See also**
+
+- [jerry_create_error](#jerry_create_error)
+
+
+# jerry_get_boolean_value
+
+**Summary**
+
+Gets the raw bool value form a `jerry_value_t` value.
+
+**Prototype**
+
+```c
+bool
+jerry_get_boolean_value (const jerry_value_t value);
+```
+
+- `value` - api value
+- return value - boolean value represented by the argument.
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  jerry_value_t value;
+  ... // create or acquire value
+
+  if (jerry_value_is_boolean (value))
+  {
+    bool raw_value = jerry_get_boolean_value (value);
+
+    ... // usage of raw value
+
+  }
+
+  jerry_release_value (value);
+}
+
+```
+
+**See also**
+
+- [jerry_value_is_boolean](#jerry_value_is_boolean)
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_get_number_value
+
+**Summary**
+
+Gets the number value of the given `jerry_value_t` parameter as a raw double.
+
+**Prototype**
+
+```c
+double
+jerry_get_number_value (const jerry_value_t value);
+```
+
+- `value` - api value
+- return value - the number value of the given `jerry_value_t` parameter as a raw double.
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  jerry_value_t value;
+  ... // create or acquire value
+
+  if (jerry_value_is_number (value))
+  {
+    double raw_value = jerry_get_number_value (value);
+
+    ... // usage of raw value
+
+  }
+
+  jerry_release_value (value);
+}
+```
+
+**See also**
+
+- [jerry_value_is_number](#jerry_value_is_number)
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_get_object_value
+
+**Summary**
+
+Gets the object value of the given `jerry_value_t` parameter.
+
+**Prototype**
+
+```c
+jerry_object_t *
+jerry_get_object_value (const jerry_value_t value);
+```
+
+- `value` - api value
+- return value - the v_object member of the given `jerry_value_t` parameter.
```

Comment:
This feels a bit too technical. Can this be something like `return value - the object contained in the given 'jerry_value_t' parameter`